2021-10-14 14:37:37,956 not enough "converse" intents were generated from templates.  Limiting number of samples to 90
2021-10-14 14:37:37,964 not enough "get_alarms" intents were generated from templates.  Limiting number of samples to 43
2021-10-14 14:37:37,964 not enough "get_date" intents were generated from templates.  Limiting number of samples to 32
2021-10-14 14:37:37,964 not enough "get_music_info" intents were generated from templates.  Limiting number of samples to 33
2021-10-14 14:37:37,964 not enough "get_time" intents were generated from templates.  Limiting number of samples to 43
2021-10-14 14:37:37,964 not enough "get_todos" intents were generated from templates.  Limiting number of samples to 41
2021-10-14 14:37:37,966 not enough "pause_music" intents were generated from templates.  Limiting number of samples to 40
2021-10-14 14:37:37,968 not enough "previous_music" intents were generated from templates.  Limiting number of samples to 41
2021-10-14 14:37:37,969 not enough "resume_music" intents were generated from templates.  Limiting number of samples to 30
2021-10-14 14:37:37,973 not enough "skip_music" intents were generated from templates.  Limiting number of samples to 51
2021-10-14 14:37:37,973 not enough "stop" intents were generated from templates.  Limiting number of samples to 61
2021-10-14 14:37:47,124 AutoGraph could not transform <bound method StreamWrapper.write of <colorama.ansitowin32.StreamWrapper object at 0x00000149FC988640>> and will run it as-is.
Cause: mangled names are not yet supported
To silence this warning, decorate the function with @tf.autograph.experimental.do_not_convert
2021-10-14 14:37:46,498 The parameters `output_attentions`, `output_hidden_states` and `use_cache` cannot be updated when calling a model.They have to be set to True/False in the config object (i.e.: `config=XConfig.from_pretrained('name', output_attentions=True)`).
2021-10-14 14:37:47,292 The parameter `return_dict` cannot be set in graph mode and will always be set to `True`.
2021-10-14 14:37:47,475 From C:\Users\trebo\miniconda3\envs\aurras\lib\site-packages\tensorflow\python\ops\array_ops.py:5043: calling gather (from tensorflow.python.ops.array_ops) with validate_indices is deprecated and will be removed in a future version.
Instructions for updating:
The `validate_indices` argument has no effect. Indices are always validated on CPU and never validated on GPU.
